ex11_ch3.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#step1: get input on amount of money in euros
+amount_of_euros = float(input("How much money in euros do you have? "))
+
+#step2: get input about the exchange rate
+euro_exchange_rate = float(input("What is the current euro exchange rate? "))
+
+#step3: write calculation to convert the euros to US dollars
+dollar_exchange_rate = 1.17
+
+conversion_euros_to_dollars = (amount_of_euros * euro_exchange_rate) / dollar_exchange_rate
+rounded_conversion = round(conversion_euros_to_dollars)
+
```

Comment:
I can do the conversion without curly braces -- why is that? Isn't it typical to use curly braces when you use the input from the user?

Reply:
You mean parentheses? And I assume you mean line 18? The PEMDAS rule in math is used by Python, which means the order of multiplication and division don't matter. But if you want to do addition/subtraction before multiplication/division, then you should use parentheses, because they have the highest precedence. The way you did it here is how I would have done it.